src/App.tsx

```diff
@@ -1,38 +1,26 @@
 import * as React from 'react';
 import './App.css'
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
 
-const list = [
-  {
-    title: 'Cleaning',
-    id: 0,
-  }, {
-    title: 'Groceries',
-    id: 1,
-  }, {
-    title: 'Work',
-    id: 2,
-  },
-  {
-    title: 'Sleep',
-    id: 3,
-  },
-];
+// @Ethan I'm going to use sudo code to discuss questions/topics that come up while i'm coding.
```

Comment:
Ethan got reassigned and I'll be reviewing your code. Not a fan of the comments in the code since I'll ask you to remove all of them before I send an approval. If you want to ask a question regarding your code, the best way to do it is by leaving a comment in the PR (like this message).

Reply:
I understand. Sometimes I come up with questions that I don't necessarily want to focus on at the moment, but rather discuss them in future conversations. I will find another way.